packages/frontend/src/utils/data-decoding.ts

```diff
@@ -7,11 +8,14 @@ interface DecodedData {
 }
 
 export const decodeOracleData = (data: string): DecodedData | null => {
-    const [realityV3Address, questionId, question] =
-        ethers.utils.defaultAbiCoder.decode(
-            ["address", "bytes32", "string"],
-            data
-        ) as [string, string, string];
+    const [realityV3Address, questionId, question] = decodeAbiParameters(
+        [
+            { type: "address", name: "realityV3Address" },
+            { typr: "bytes32", name: "question" },
+            { type: "string", name: "questionId" },
+        ],
+        data as Address
+    ) as [string, string, string];
```

Comment:
Are we sure this is right? `decodeAbiParameters` should already return sensible types

Reply:
Fixed in f44ffd9afee046bbfeeca0ecb66ab8586fee4e83.